File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/database/model/RealmLocation.java
package com.lattis.ellipse.data.database.model;

import io.realm.RealmObject;

public class RealmLocation extends RealmObject{

    private double latitude;
    private double longitude;

    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    @Override
    public String toString() {
        return "RealmLocation{" +
                "latitude=" + latitude +
                ", longitude=" + longitude +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/database/model/RealmLocation.kt
package com.lattis.data.database.model

import io.realm.RealmObject

open class RealmLocation : RealmObject() {
    var latitude = 0.0
    var longitude = 0.0

    override fun toString(): String {
        return "RealmLocation{" +
                "latitude=" + latitude +
                ", longitude=" + longitude +
                '}'
    }
}